docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -123,15 +123,18 @@ receive requests from NGINX instance.
 It is set to ``80`` by default.
 
 
-vars.bigchaindb-api-port and vars.bigchaindb-ws-port
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+vars.bigchaindb-api-port, vars.bigchaindb-ws-port and Similar
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The ``bigchaindb-api-port`` is the port number on which BigchainDB is
 listening for HTTP requests. Currently set to ``9984`` by default.
 
 The ``bigchaindb-ws-port`` is the port number on which BigchainDB is
 listening for Websocket requests. Currently set to ``9985`` by default.
 
+Other ``bigchaindb-*`` parameters are used to configure BigchainDB instance.
+The complete list can be found `here <https://docs.bigchaindb.com/projects/server/en/latest/server-reference/configuration.html>`_.
```

Comment:
We should try to avoid linking to URLs like that, preferring references instead. Also, a more descriptive name for the linked-page helps Google with indexing. How about:
```text
There's another :ref:`page with a complete listing of all BigchainDB Server configuration settings <Configuration Settings>`.
```